src/components/Carousel/index.js

```diff
@@ -75,26 +75,63 @@ export default class Carousel extends PureComponent {
     highlightOnHover: false,
     loop: false,
     overflow: false,
-    scrollCount: 1,
+    scrollCount: 2,
     showCount: 3,
     transition: TRANSITION_SLIDE,
   }
 
+  state = {
+    currentSlide: 0,
```

Comment:
Will this override the prop that lets you pass in what slide you want to start on? I know we've passed in custom slide starting values in the past

Reply:
good catch